web/source/keyboards/kmwkeyboards.ts

```diff
@@ -119,8 +119,8 @@ namespace com.keyman.keyboards {
         if (language.region) {
           stub['KR'] = language.region;
         }
-        // RegionCode? 
-        // stub['KRC'] = language.???;
+
+        // Can ignore ['KRC'] RegionCodes used by the Toolbar UI
```

Comment:
Just to be safe, do make sure to test one round with the Toolbar UI test page, as that's what the region codes are for.

Reply:
Good suggestion. I ran the toolbar test page and found an exception from  `kmw.addKeyboards()` I'll have to fix

Reply:
The /web/testing/desktop-ui/toolbar.html test page is working now.

Reply:
Hmm.  Looks like that's mostly because `KRC` isn't really used at this point.  I don't see any references to it from within `kmwuitoolbar.ts`.